configuration_version.go

```diff
@@ -185,6 +186,15 @@ func (s *configurationVersions) Read(ctx context.Context, cvID string) (*Configu
 func (s *configurationVersions) Upload(ctx context.Context, url, path string) error {
 	body := bytes.NewBuffer(nil)
 
+	file, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
+
+	if !file.Mode().IsDir() {
+		return errors.New("Path needs to be a directory")
```

Comment:
Maybe make this `Path needs to be an existing directory` to make it even more specific?